tests/integration/envrc.rs

```diff
@@ -76,3 +76,21 @@ fn v2_gopath_previously_unset() {
 
     assert_eq!(env.get_env("GOPATH"), DirenvValue::Value("BAR"));
 }
+
+#[test]
+fn bug18_path_with_spaces() {
```

Comment:
What does the "18" refer to here?

Reply:
There seemed to be a naming convention for regression tests, so this refers to issue #18